src/buffer.rs

```diff
@@ -708,14 +708,16 @@ mod test {
 
     #[test]
     fn test_array_buffer() {
+        use assert_approx_eq::assert_approx_eq;
```

Comment:
In this case we actually want to check for exact equality to ensure that we didn't loose precision from roundtrip to python.